src/Scripts/Character_Scripts/Character.gd

```diff
@@ -114,8 +114,14 @@ func ui_accept_pressed():
   if(interact_areas.size() != 0 and interact_areas.back() != ""):
 	  DialogueEngine._beginTransmit(interact_areas.back())
 	
-func change_sequenced_follow_formation(formation: String):
+func change_follow(formation: String):
 	self.party_data["sequence_formation"] = formation
+	
+func change_speed(speed_string: String):
+	var default_speed: float = 60
+	var new_speed = float(speed_string)
+	$AnimatedSprite.set_speed_scale(new_speed/default_speed) 
+	self.speed = float(new_speed)
```

Comment:
Should also have a resotre_default_speed() function, so it can be generically restored without knowing what an entities default speed is.